index.js

```diff
@@ -9,11 +9,17 @@ const debuglog = util.debuglog('aperture');
 
 class Aperture {
   constructor() {
-    macosVersion.assertGreaterThanOrEqualTo('10.10');
+    if (process.platform === 'darwin') {
```

Comment:
Create top level constants IS_MACOS and IS_LINUX to simplify all these if else statements.